includes/Core/User_Input/User_Input.php

```diff
@@ -67,15 +67,18 @@ class User_Input {
 	 * @var array|ArrayAccess
 	 */
 	private static $questions = array(
-		'purpose'       => array(
+		'purpose'                 => array(
 			'scope' => 'site',
 		),
-		'postFrequency' => array(
+		'postFrequency'           => array(
 			'scope' => 'user',
 		),
-		'goals'         => array(
+		'goals'                   => array(
 			'scope' => 'user',
 		),
+		'includeConversionEvents' => array(
```

Comment:
Setting which stores the conversion events during the tailored metrics setup or site purpose answer change, has been moved to the user input, within the site scope, so view only users and other users can have access to the conversion reporting metrics after first admin setup tailored metrics